HttpSignature.md

```diff
@@ -91,17 +91,14 @@ located elsewhere on the web, such as the requestor's [Freedom Box](https://free
 
 ```HTTP
 Signature-Input: sig1=(); keyId="<https://alice.freedombox/keys/test-key-a>"; created=1402170695
-Signature: sig1=:cxieW5ZKV9R9A70+Ua1A/1FCvVayuE6Z77wDGNVFSiluSzR9TYFV
-       vwUjeU6CTYUdbOByGMCee5q1eWWUOM8BIH04Si6VndEHjQVdHqshAtNJk2Quzs6WC
-       2DkV0vysOhBSvFZuLZvtCmXRQfYGTGhZqGwq/AAmFbt5WNLQtDrEe0ErveEKBfaz+
-       IJ35zhaj+dun71YZ82b/CRfO6fSSt8VXeJuvdqUuVPWqjgJD4n9mgZpZFGBaDdPiw
-       pfbVZHzcHrumFJeFHWXH64a+c5GN+TWlP8NPg2zFdEc/joMymBiRelq236WGm5VvV
-       9a22RW2/yLmaU/uwf9v40yGR/I1NRA==:
+Signature: sig1=:cxieW5ZKV9R9A70+Ua1A/1FCvVayuE6Z77wDGNVFSiluSz...==:
 ```
 
+It could also allow key based did URLs as described in [issue 217 of the Solid Spec](https://github.com/solid/specification/issues/217#issuecomment-777509084).
+
 The advantage of a URL is that it allows the client to use HTTP Methods such as `POST` or `PUT` to create keys, as well as `PUT`, `PATCH` and `DELETE` to edit them, solving the problem of key revocation.
 
-We also reserve the use of keyIds enclosed with `>` and `<` characters for
+We also reserve the use of keyIds enclosed with `'>'` and `'<'` characters for
```

Comment:
This suggests `>{keyId}<` which I think is not what is intended.

"Enclosed with" usually enumerates first the prepend and second the append -- so if the intent is (as I believe) `<{keyId}>`, this should read 
```suggestion
We also reserve the use of keyIds enclosed with `'<'` and `'>'` characters for
```

Reply:
`>{keyId}<` is exactly what was intended! :-) 

The idea is that we want a way to say: this is the relative URL for the key, but it is not to be found out there on the internet, but by asking the client for it. 
How does one ask the client for it? By using the P2P extension of HTTP/2 referred to, which allows the server to take on the role of a client on the same connection. 

Do you think `>{keyId}<` would be clearer?